### PR TITLE
Cart Page

### LINKED ISSUE #17: Cart-page
- addToCart functionality from products page.
> Cart gets updated in User context.
- bulit ui of  cart-page in react.

### PR DESCRIPTION
**Pull request by tushartiwari7**
Add To Cart when user clicks add To Cart btn in product card.
It fires the api to `/api/user/cart` which returns the updated Cart and also updates cart in Backend.
Then User State's `cart` property gets updated with the returned cart.

### Major Commit
[feat: AddToCart from Product Page](https://github.com/tushartiwari7/furn-house/commit/577d305447e6d3710dd973d5698da5d85b10073f)
> The implementation of above mentioned pseudo-code
### Commits You Can Ignore
[Cart-Page JSX and CSS](https://github.com/tushartiwari7/furn-house/commit/92a1a0aaf4cabe8160fa8ab1c531613d27360324)

### CODE REVIEW

File: src/context/user/UserContext.jsx [hunk 1]
```diff
@@ -6,6 +6,23 @@ const UserContext = createContext();
 export const UserProvider = ({children}) => {
 
   const [user,setUser] = useState({isLoggedIn: false});
+
+  /**
```
Comment (ShreyashMahajan) [inline src/context/user/UserContext.jsx:10]: commented code can be removed

**Review by ShreyashMahajan — APPROVED** [review]
code looks well strucutured and clean. Good to merge

File: src/components/CartItem/CartItem.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+              -
+            </button>
+            <input
+              className="text-center rounded-circle mx-xs"
+              type="number"
+              defaultValue={1}
+              min={1}
+            />
+            <button className="btn rounded-circle btn-outline-primary">
+              +
```
Comment (JantuDeb) [inline src/components/CartItem/CartItem.jsx]: Use icon for +,-

**Review by JantuDeb — APPROVED** [review]
This looks good

File: src/context/user/UserContext.jsx (hunk 1 above)
Comment (tushartiwari7) [reply src/context/user/UserContext.jsx:10]: This was intentionally put to give a view how my user data will look after login.

File: src/components/CartItem/CartItem.jsx (hunk 2 above)
Comment (tushartiwari7) [reply src/components/CartItem/CartItem.jsx]: done

**Pull request merged**